packages/core/server-utils/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@parcel/server-utils",
```

Comment:
Should this be a separate package or just part of @parcel/utils?

Reply:
I don't mind adding it to the utils